src/textures/Source.js

```diff
@@ -81,6 +87,10 @@ class Source {
 
 			target.set( data.videoWidth, data.videoHeight );
 
+		} else if ( data instanceof VideoFrame ) {
+
+			target.set( data.displayHeight, data.displayWidth );
```

Comment:
Applying the same change in `Source.getSize()` to make sure the `width`, `height` and `depth` accessors in texture work as expected.